cli/lib/cli.js

```diff
@@ -154,26 +154,16 @@ const text = (description) => {
 
 function includesVersion (args) {
   return (
-    _.includes(args, 'version') ||
     _.includes(args, '--version') ||
     _.includes(args, '-v')
   )
 }
 
-function showVersions (args) {
+function showVersions (opts) {
   debug('printing Cypress version')
-  debug('additional arguments %o', args)
+  debug('additional arguments %o', opts)
 
-  const versionParser = commander.option(
-    '--component <package|binary|electron|node>', 'component to report version for',
-  )
-  .allowUnknownOption(true)
-  const parsed = versionParser.parse(args)
```

Comment:
With how this was implemented to handle both `cypress version --component XXX` and `cypress --version --component XXX`, command v6 was throwing errors of command clashing because this adds support for `--component` at the top-level, where as its an option for the `run` and `open` commands.